LICENSE-binary

```diff
@@ -0,0 +1,532 @@
+hadoop-tools/hadoop-sls/src/main/html/css/bootstrap-responsive.min.css
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/dt-1.10.18/*
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/jquery
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/jt/jquery.jstree.js
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/resources/TERMINAL
+
+bootstrap v3.3.6
+broccoli-asset-rev v2.4.2
+broccoli-funnel v1.0.1
+datatables v1.10.8
+em-helpers v0.5.13
+em-table v0.1.6
+ember v2.2.0
+ember-array-contains-helper v1.0.2
+ember-bootstrap v0.5.1
+ember-cli v1.13.13
+ember-cli-app-version v1.0.0
+ember-cli-babel v5.1.6
+ember-cli-content-security-policy v0.4.0
+ember-cli-dependency-checker v1.2.0
+ember-cli-htmlbars v1.0.2
+ember-cli-htmlbars-inline-precompile v0.3.1
+ember-cli-ic-ajax v0.2.1
+ember-cli-inject-live-reload v1.4.0
+ember-cli-jquery-ui v0.0.20
+ember-cli-qunit v1.2.1
+ember-cli-release v0.2.8
+ember-cli-shims v0.0.6
+ember-cli-sri v1.2.1
+ember-cli-test-loader v0.2.1
+ember-cli-uglify v1.2.0
+ember-d3 v0.1.0
+ember-data v2.1.0
+ember-disable-proxy-controllers v1.0.1
+ember-export-application-global v1.0.5
+ember-load-initializers v0.1.7
+ember-qunit v0.4.16
+ember-qunit-notifications v0.1.0
+ember-resolver v2.0.3
+ember-spin-spinner v0.2.3
+ember-truth-helpers v1.2.0
+jquery v2.1.4
+jquery-ui v1.11.4
+loader.js v3.3.0
+momentjs v2.10.6
+qunit v1.19.0
+select2 v4.0.0
+snippet-ss v1.11.0
+spin.js v2.3.2
+
+com.microsoft.azure:azure-cosmosdb:2.4.5
+com.microsoft.azure:azure-cosmosdb-commons:2.4.5
+com.microsoft.azure:azure-cosmosdb-direct:2.4.5
+com.microsoft.azure:azure-cosmosdb-gateway:2.4.5
+com.microsoft.azure:azure-data-lake-store-sdk:2.3.3
+com.microsoft.azure:azure-keyvault-core:1.0.0
+com.microsoft.sqlserver:mssql-jdbc:6.2.1.jre7
+org.bouncycastle:bcpkix-jdk15on:1.60
+org.bouncycastle:bcprov-jdk15on:1.60
+org.checkerframework:checker-qual:2.5.2
+org.codehaus.mojo:animal-sniffer-annotations:1.17
+org.jruby.jcodings:jcodings:1.0.13
+org.jruby.joni:joni:2.1.2
+org.slf4j:jul-to-slf4j:jar:1.7.25
+org.ojalgo:ojalgo:43.0:compile
+org.slf4j:jul-to-slf4j:1.7.25
+org.slf4j:slf4j-api:1.7.25
+org.slf4j:slf4j-log4j12:1.7.25
+
+
+Common Development and Distribution License (CDDL) 1.0
+------------------------------------------------------
+
+javax.ws.rs:jsr311-api:1.1.1
+
+
+Common Development and Distribution License (CDDL) 1.1
+------------------------------------------------------
+
+com.sun.jersey:jersey-client:1.19
+com.sun.jersey:jersey-core:1.19
+com.sun.jersey:jersey-guice:1.19
+com.sun.jersey:jersey-json:1.19
+com.sun.jersey:jersey-server:1.19
+com.sun.jersey:jersey-servlet:1.19
+com.sun.xml.bind:jaxb-impl:2.2.3-1
+javax.xml.bind:jaxb-api:2.2.11
+
+
+Eclipse Public License 1.0
+--------------------------
+
+junit:junit:4.12
+
+
+Eclipse Public License 2.0
```

Comment:
should be CDDL + GPLv2 with classpath exception?

Reply:
Yes. Thank you for your comment.
The libraries have been moved to eclipse repository and now the license is EPL, however, Apache Hadoop is using the old version of the libraries and the licenses were CDDL + GPLv2 with classpath exception.